notebooks/image_models/solutions/deepconv_gan.ipynb

```diff
@@ -50,33 +50,6 @@
     "## Import TensorFlow and other libraries"
```

Comment:
<p>Remove the Ipython import which is no longer used</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/asl-ml-immersion/pull/221/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='6'/>

Reply:
<p>Actually <code>IPython.display</code>  module is used in <code>train</code> function below. </p>

Reply:
<p>Actually <code>IPython.display</code> is used in <code>train</code> function. Do you think we can keep it?  </p>